modules/javafx.controls/src/main/java/javafx/scene/control/TableColumnBase.java

```diff
@@ -726,42 +726,42 @@ public final T getCellData(final S item) {
      */
     public abstract ObservableValue<T> getCellObservableValue(S item);
 
-    /** {@inheritDoc} */
-    @Override public EventDispatchChain buildEventDispatchChain(EventDispatchChain tail) {
+    @Override
+    public EventDispatchChain buildEventDispatchChain(EventDispatchChain tail) {
         return tail.prepend(eventHandlerManager);
     }
 
     /**
-     * Registers an event handler to this table column. The TableColumnBase class allows
-     * registration of listeners which will be notified when editing occurs.
-     * Note however that TableColumnBase is <b>not</b> a Node, and therefore no visual
-     * events will be fired on it.
-     *
-     * @param <E> The type of event
-     * @param eventType the type of the events to receive by the handler
-     * @param eventHandler the handler to register
-     * @throws NullPointerException if the event type or handler is null
+     * {@inheritDoc}
+     * <p>
+     * The {@code TableColumnBase} class allows registration of listeners which will be notified when editing occurs.
+     * Note that {@code TableColumnBase} is <b>not</b> a {@link Node}, and therefore no visual events will be fired on it.
```

Comment:
Isn't this true for the other classes here as well, like `Tab`? Might it be worth saying this in the other classes too then, or even on `EventTarget` itself (something like "Note that implementing classes that are not a `Node` don't receive visual events")?
I would also give an example or 2 of what "visual events" are since this term is not defined anywhere that I know of.

Reply:
I don't know what "visual events" are, maybe input events?
Anyway, the added value of this entire javadoc seems to be rather low, I'm inclined to just remove it entirely.

By the way, there's a great documentation of the JavaFX event system at https://docs.oracle.com/javase/8/javafx/events-tutorial/events.htm. It would be really helpful if we could add this to the JavaFX codebase in the future.

Reply:
> I don't know what "visual events" are, maybe input events?
I thought something like that too, including scroll and touch events. Maybe @kevinrushforth knows.

> Anyway, the added value of this entire javadoc seems to be rather low, I'm inclined to just remove it entirely.
I don't think I mind removing it. My more general concern is that it's not obvious what events can be registered on different `EventTarget`s.

> By the way, there's a great documentation of the JavaFX event system at https://docs.oracle.com/javase/8/javafx/events-tutorial/events.htm. It would be really helpful if we could add this to the JavaFX codebase in the future.

Yes, I had an idea to link to the tutorials when working on controls documentation (https://bugs.openjdk.org/browse/JDK-8252550). The problem is that we don't have access to them and can't update them if they are out of date. Embedding them is probably too verbose. Linking looks like the better option.

Reply:
I've removed the documentation from `TableColumnBase` and instead added a paragraph to the class documentation of both `TableColumn` and `TreeTableColumn`.